File: src/com/gmail/marcosav2010/logger/BaseLog.java
package com.gmail.marcosav2010.logger;

import com.gmail.marcosav2010.logger.Logger.VerboseLevel;

public class BaseLog implements ILog {

    @Override
    public void log(Object o) {
        Logger.log(o);
    }

    @Override
    public void log(Object o, VerboseLevel level) {
        Logger.log(o, level);
    }

    @Override
    public void log(Throwable t) {
        Logger.log(t);
    }

    @Override
    public void log(Throwable t, String msg) {
        Logger.log(t, msg);
    }
}

File: core/src/main/java/com/gmail/marcosav2010/logger/BaseLog.kt
package com.gmail.marcosav2010.logger

import com.gmail.marcosav2010.logger.Logger.VerboseLevel

class BaseLog internal constructor() : ILog {

    override fun log(o: Any?) = Logger.log(o)

    override fun log(o: Any?, level: VerboseLevel) = Logger.log(o, level)

    override fun log(t: Throwable) = Logger.log(t)

    override fun log(t: Throwable, msg: String?) = Logger.log(t, msg)
}